tests/cmaize/user_api/dependencies/test_find_dependency.cmake

```diff
@@ -0,0 +1,65 @@
+# Copyright 2023 CMakePP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+include(cmake_test/cmake_test)
+include(cmaize/user_api/cmaize_project)
+include(cmaize/user_api/dependencies/find_dependency)
+include(cmaize/user_api/dependencies/impl_/get_package_manager)
+include(cmaize/utilities/python)
+
+ct_add_test(NAME "cmaize_find_dependency")
+function("${cmaize_find_dependency}")
+
+    # Create a project
+    set(proj_name "test_find_dependency")
+    project("${proj_name}")
+    cmaize_project("${proj_name}")
+    cpp_get_global(proj_obj CMAIZE_TOP_PROJECT)
+
+    find_python(py_exe py_version)
+    create_virtual_env(
+        venv_dir "${py_exe}" "${CMAKE_BINARY_DIR}" "${_find_dependency}"
```

Comment:
```suggestion
        venv_dir "${py_exe}" "${CMAKE_BINARY_DIR}" "${cmaize_find_dependency}"
```

Reply:
Maybe I should have quoted more of the code for this one. The test case set up above is `cmaize_find_dependency`:
```
ct_add_test(NAME "cmaize_find_dependency")
function("${cmaize_find_dependency}")
```